bin/validate_lastnight_fba

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# requires source /project/projectdirs/desi/software/desi_environment.sh master
```

Comment:
/global/cfs/cdirs/desi/software/desi_environment.sh (/project/projectdirs alias exists on cori but not on perlmutter)